src/libraries/System.Private.CoreLib/src/System/Threading/TimeoutHelper.cs

```diff
@@ -21,12 +21,20 @@ public static uint GetTime()
             return (uint)Environment.TickCount;
         }
 
+        /// <summary>
+        /// Returns <see cref="Environment.TickCount64"/> as a start time in milliseconds.
+        /// </summary>
+        public static long GetTime64()
+        {
+            return Environment.TickCount64;
```

Comment:
I do not see the value of wrapping ` Environment.TickCount64` in a helper function. It is probably left over from early days of .NET Core where we had fine-grained layering.


Delete this method (and the existing GetTime() method) and call Environment.TIckCount64 directly?